test/dlc_tests/eks/eks_manifest_templates/kubeflow/install_kubeflow_custom_kfctl.sh

```diff
@@ -0,0 +1,75 @@
+#!/bin/bash
+set -e
+
+install_kfctl(){
+    #install kfctl cli
+    KFCTL_VERSION="v1.0.2"
+    S3_BUCKET="kubeflow-kfctl-binary"
+
+    if ! command -v kfctl &> /dev/null
+    then
+        aws s3 cp s3://${S3_BUCKET}/kfctl_${KFCTL_VERSION}-1-g93e95e1_linux.tar.gz /tmp/kfctl_${KFCTL_VERSION}_linux.tar.gz
```

Comment:
Add
```
echo "not installed"
```
as it was in the original file, as well.

Reply:
this statement needs to be removed. PR for the same #541